tokio/src/sync/watch.rs

```diff
@@ -112,6 +112,56 @@ pub struct Sender<T> {
 #[derive(Debug)]
 pub struct Ref<'a, T> {
     inner: RwLockReadGuard<'a, T>,
+    has_changed: bool,
+}
+
+impl<'a, T> Ref<'a, T> {
+    /// Indicates if the borrowed value is considered as _changed_ since the last
+    /// time it has been marked as seen.
+    ///
+    /// Other than [`Receiver::has_changed()`] it does not fail if the [`Receiver`]
+    /// is orphaned after the [`Sender`] has been dropped.
```

Comment:
What do you mean by this?

Reply:
`Receiver::has_changed()` fails with an error if the channel is considered as "closed". This happens when the `Sender` is dropped. Which doesn't makes much sense to me, because the shared value (and therefore the "channel"'s content) is still accessible.

I am not a native speaker. If this needs to be reworded please do so.

Reply:
```suggestion
    /// Unlike [`Receiver::has_changed()`], this method does not fail if the channel is closed.
```